python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -2844,8 +2848,7 @@ def test_cuda_array_interface(dtype):
 @pytest.mark.parametrize("data_type", dtypes)
 def test_from_arrow_chunked_arrays(nelem, nchunks, data_type):
     np_list_data = [
-        np.random.randint(0, 100, nelem).astype(data_type)
-        for i in range(nchunks)
+        rng.integers(0, 100, nelem).astype(data_type) for i in range(nchunks)
     ]
```

Comment:
*nit*: Make an `rng` here, rather than relying on the global one?

Reply:
Done.